### PR TITLE
Update vuln list schema and response

### PR DESCRIPTION
**Pull request by Zhepu-zingbox**
## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes if appropriate.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Zhepu-zingbox** [comment]
@csestito Could u help review and merge the PR? Thx!

File: openapi-specs/iot/api/public/alertResolve.yaml [hunk 1]
```diff
@@ -37,7 +38,7 @@ requestBody:
           value:
             reason: The alert poses no threat
             resolved: yes
-            reason_type: ['No Action Needed']
+            reason_type: ['Issue Mitigated']
```
Comment (sserrata) [inline openapi-specs/iot/api/public/alertResolve.yaml:41]: This appears to be type Array in this example but type `string` in the documentation.

**Review by sserrata — COMMENTED** [review]
See comment on "alert resolve" endpoint

**Review by shuailiu-zingbox — APPROVED** [review]

**Review by sserrata — APPROVED** [review]
Approving but please note the inconsistency between the "Alert Resolve" schema and example. Thanks!

File: openapi-specs/iot/api/public/alertResolve.yaml (hunk 1 above)
Comment (Zhepu-zingbox) [reply openapi-specs/iot/api/public/alertResolve.yaml:41]: > This appears to be type Array in this example but type `string` in the documentation.

great catch! schema updated.

**Comment by Zhepu-zingbox** [comment]
> Approving but please note the inconsistency between the "Alert Resolve" schema and example. Thanks!

comment fixed! Could u help merge it if looks good to you?

**Pull request merged**